packages/btcindexer/src/api/client.ts

```diff
@@ -38,4 +38,45 @@ export default class Client {
 		}
 		return response.json();
 	}
+
+	async postNbtcTx(
+		txHex: string,
```

Comment:
I think the function should take the bearer token as an argument for authentication, and then set it in the headers

Reply:
Or , (even better) - let's set the token in the constructor.